content/de/docs/concepts/containers/images.md

```diff
@@ -299,9 +300,9 @@ resources:
 EOF
 ```
 
-Dies muss für jeden Pod getan werden der eine eigene Registry nutzt.
+Dies muss für jeden Pod getan werden, der eine eigene Registry nutzt.
```

Comment:
/assign
@rezemble
pls see https://git.k8s.io/community/CLA.md#the-contributor-license-agreement

Reply:
/LGTM